pages/auth/TEMPLATE.js

```diff
@@ -0,0 +1,34 @@
+import Head from "next/head";
```

Comment:
This is a template for all pages going forward.  This is because a user shouldn't have access to any pages if they are not authenticated meaning we need to wrap each page in firebase authentication checks.  There might be an easier way to do this but this is what I have come up with.  Everything is the same inside the `return()` so there should be no problems